.ci/setup.sh

```diff
@@ -93,6 +93,9 @@ install_extra_tools() {
 	echo "Install CNI plugins"
 	bash -f "${cidir}/install_cni_plugins.sh"
 
+	# Remove K8s + CRIO conf that may remain from a previous run
+	rm -f /etc/systemd/kubelet.service.d/0-crio.conf
```

Comment:
missing `/system/` ?  `0-crio.conf` looks very specific, what about removing whole `kubelet.service.d` directory?

Reply:
Yep, system was missing, committed the wrong file.

Not sure about deleting the whole directory, on my system (Ubuntu 20.04 ppc64) there is a 10-kubeadm.conf file in this directory, that came with the kubeadm package.